### DISCUSSION TITLE
IPFS 的公共网关节点可以发一下吗？

### DISCUSSION THREAD

**Discussion by 0x1992** [filecoin-project/community-china#96] [应用开发]
用来请求文件内容的公共网关节点，可以帮忙发出来一些吗？有收集公共网关的网站也行。非常感谢！

**Comment by The-Wayvy** [comment]
您好0x1992 
[公共网关节点的列单](https://ipfs.github.io/public-gateway-checker/)
您还需要进一步的帮助吗
